example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -138,8 +136,12 @@ struct FillBufferKernel
 auto main()
 -> int
 {
+// This example is hard-coded to use the sequential executor.
+// It can not be compiled if it is not available.
```

Comment:
Sorry for being picky about English again. I think this sentence could be improved: 'it' is used two times for different things, and also the point of this commit is so that the test actually compiles now but does nothing. Maybe the first sentence is enough and the second can be just removed?

Reply:
I will remove the second line.

Reply:
done